src/plugins/http/gfal_http_copy.cpp

```diff
@@ -136,7 +136,7 @@ static void set_copy_mode_from_urls(gfal2_context_t context, const char * src_ur
 
 static bool is_http_scheme(const char* url)
 {
-    const char *schemes[] = {"http:", "https:", "dav:", "davs:", "s3:", "s3s:", "gcloud:", "gclouds:", "swift:", "swifts:", NULL};
+    const char *schemes[] = {"http:", "https:", "dav:", "davs:", "s3:", "s3s:", "gcloud:", "gclouds:", "swift:", "swifts:", "cs3:", NULL};
```

Comment:
If you're adding `cs3s` in [Davix](https://github.com/cern-fts/davix/pull/72#discussion_r700010963), make sure to add it also here.